Misc/NEWS.d/next/IDLE/2019-02-23-17-53-53.bpo-36096.mN5Ly3.rst

```diff
@@ -0,0 +1 @@
+Refactor class variables to instance variables in colorizer.
```

Comment:
For the present, I will take care of the idlelib news.txt entries.

Reply:
I should have asked this earlier, but once you approve a PR, should I then merge it?  Thanks.

Reply:
If I check the []Approve box for a Review, and if you agree that it is ready, and accept the responsibility of doing the merge, yes.  For most IDLE issues though, I consider the followup responsibility to be shared.  If you prefer me to merge, say so.  If a PR is actually a joint product, or if it is blocking doing another, I might merge it, but should say so first.

Reply:
Thank you, that's very helpful.  I think this one looks OK and you've approved it, so I'll merge it soon.